frontend/pages/hosts/details/cards/Software/SelfService/_styles.scss

```diff
@@ -81,6 +91,34 @@
     }
   }
 
+  /* Ensures table doesn't bleed over 100% */
+  .data-table__table {
+    table-layout: fixed;
+    width: 100%;
+  }
+
+  /* Override any defaults for fixed column width */
+  .data-table__table th.ui_status__header,
+  .data-table__table td.ui_status__cell {
+    width: 140px;
+    max-width: 140px;
+    min-width: 140px;
```

Comment:
Non-blocking: why are we having to provide min/max in addition to an explicitly defined width/

Reply:
To override any `.data-table__table td` min-width / max-width that might already be lurking from the code base (e.g. I believe we have some `.data-table__table td` set to max-width: 500px)

Reply:
cc: @iansltx